### PR TITLE
Update Synology Documentation

### PR DESCRIPTION
**Pull request by cniweb**
Fixes:
https://github.com/openhab/openhab-syno-spk/issues/80
https://github.com/openhab/openhab-syno-spk/issues/52

Signed-off-by: Christian Häussler <c-n-i@web.de> (github: @cniweb)

### CODE REVIEW

File: installation/synology.md [hunk 1]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
...
-|-------------|-----------------|
-| DS209       | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0.009-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.009-SNAPSHOT-DSM6) |
-| DS216play   | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS413j      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS916+      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS214play   | [2.1.0.004-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.1.0.004-SNAPSHOT-DSM6) |
-| DS1815+     | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
+If no public shared folder exists, the OpenHAB install script will place configuration files are located at
```
Comment (ThomDietrich) [inline installation/synology.md]: *openHAB

File: installation/synology.md [hunk 2]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
...
-| DiskStation | Package version |
-|-------------|-----------------|
-| DS209       | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0.009-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.009-SNAPSHOT-DSM6) |
-| DS216play   | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS413j      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS916+      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS214play   | [2.1.0.004-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.1.0.004-SNAPSHOT-DSM6) |
-| DS1815+     | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
```
Comment (ThomDietrich) [inline installation/synology.md]: What happens to the list?

File: installation/synology.md [hunk 3]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
 
 ## Configuration
 
-The configuration files are located at:
+The location of the configuration files varies depending if a shared folder named public exists. The public folder is created automatically after your Synology NAS is first set up for certain models. The administrator can also create a public shared folder if desired. Synology does not document what models will automatically create a public folder. See [DiskStation Manager - Shared Folder](https://www.synology.com/en-us/knowledgebase/DSM/help/DSM/AdminCenter/file_share_desc)
 
-```conf
-/volume1/@appstore/openHAB2/userdata/config
-```
+If a public shared folder exists, the OpenHAB install script will place configuration files are located at:
```
Comment (ThomDietrich) [inline installation/synology.md]: *openHAB

File: installation/synology.md [hunk 4]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
 
 ## Configuration
 
-The configuration files are located at:
+The location of the configuration files varies depending if a shared folder named public exists. The public folder is created automatically after your Synology NAS is first set up for certain models. The administrator can also create a public shared folder if desired. Synology does not document what models will automatically create a public folder. See [DiskStation Manager - Shared Folder](https://www.synology.com/en-us/knowledgebase/DSM/help/DSM/AdminCenter/file_share_desc)
```
Comment (ThomDietrich) [inline installation/synology.md]: Please break down to one sentence per line

Comment (ThomDietrich) [inline installation/synology.md]: > if a shared folder named "public" exists
or
> if a shared folder named `public` exists

*which

File: installation/synology.md [hunk 5]
```diff
@@ -21,6 +21,16 @@ java.security.NoSuchAlgorithmException: EC KeyPairGenerator not available
...
+
+**Armada 370/XP** systems running DSM 6 require hard float ABI rather than the soft float used on DSM 5. If there is not a Java8 package in the Package Center, your machine may not be supported by the [Synology package](https://www.synology.com/en-us/dsm/packages/Java8).
+
+The best option for older unsupported Synology models such as **PowerPC** and **ARJava SEM** is to include the *PC-Loadletter* Repo. [Download Java SE](http://tinyurl.com/javaembed) (Java Cryptography Extension if required) and put them in the *public* folder of your NAS. Then start the installation of Java8 SE.
+
+*Note that it may be necessary to rename the downloaded file to match the name the Loadletter utility expects which may be an older version than what is currently available.*
+
+The **Java8 PowerPC** versions are available from [Oracle](http://www.oracle.com/technetwork/java/embedded/embedded-se/downloads/javaseembedded8u6-2406243.html)
```
Comment (ThomDietrich) [inline installation/synology.md]: Please break down to one sentence per line

Comment (ThomDietrich) [inline installation/synology.md]: Also in this sentence the ending . is missing

File: installation/synology.md [hunk 6]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
...
-The configuration files are located at:
+The location of the configuration files varies depending if a shared folder named public exists. The public folder is created automatically after your Synology NAS is first set up for certain models. The administrator can also create a public shared folder if desired. Synology does not document what models will automatically create a public folder. See [DiskStation Manager - Shared Folder](https://www.synology.com/en-us/knowledgebase/DSM/help/DSM/AdminCenter/file_share_desc)
 
-```conf
-/volume1/@appstore/openHAB2/userdata/config
-```
+If a public shared folder exists, the OpenHAB install script will place configuration files are located at:
 
-## Tested Devices
+`/volume1/public/openHAB2/conf/`
```
Comment (ThomDietrich) [inline installation/synology.md]:
```conf
    /volume1/public/openHAB2/conf/
    ```

File: installation/synology.md [hunk 7]
```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
...
-| DS216+II    | [2.0.0.009-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.009-SNAPSHOT-DSM6) |
-| DS216play   | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS413j      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS916+      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS214play   | [2.1.0.004-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.1.0.004-SNAPSHOT-DSM6) |
-| DS1815+     | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
+If no public shared folder exists, the OpenHAB install script will place configuration files are located at
 
-> Please complete the list accordingly, thank you!
+`/volume1/@appstore/openHAB2/conf/`
```
Comment (ThomDietrich) [inline installation/synology.md]:
```conf
    /volume1/@appstore/openHAB2/conf/
    ```

File: installation/synology.md (hunk 2 above)
Comment (cniweb) [inline installation/synology.md]: The list is constantly changing and we are currently developing a package for all DiskStation models.

**Comment by cniweb** [comment]
Hello @ThomDietrich I hope I have now implemented all your comments.

**Comment by ThomDietrich** [comment]
I've moved around a few sentences to improve the read flow and the presentation.

Thanks!

**Pull request merged**